docs/user-manual/security.adoc

```diff
@@ -13,7 +28,6 @@ The size of the caches are controlled by the `authentication-cache-size` and `au
 Both default to `1000`.
 
 How long cache entries are valid is controlled by `security-invalidation-interval`, which is in milliseconds.
-Using `0` will disable caching.
```

Comment:
Should this have moved up to the cache-size entries above, rather than being removed entirely?

Reply:
Done.